netjsonconfig/backends/base/backend.py

```diff
@@ -126,15 +127,15 @@ def _deduplicate_files(self):
             files_dict[file['path']] = file
         self.config['files'] = list(files_dict.values())
 
-    @draft4_format_checker.checks('cidr', AssertionError)
+    @format_checker.checks('cidr', AssertionError)
```

Comment:
Adding context from the [release notes of jsonschema 4.16.0](https://github.com/python-jsonschema/jsonschema/blob/main/CHANGELOG.rst#v4160):

> Accessing jsonschema.draftN_format_checker is deprecated. Instead, if you want access to the format checker itself, it is exposed as jsonschema.validators.DraftNValidator.FORMAT_CHECKER on any jsonschema.protocols.Validator.